multisrc/overrides/mangathemesia/manhwafreakfr/src/ManhwaFreakFR.kt

```diff
@@ -0,0 +1,32 @@
+package eu.kanade.tachiyomi.extension.fr.manhwafreakfr
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.SChapter
+import okhttp3.Request
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ManhwaFreakFR : MangaThemesia("ManhwaFreak.fr", "https://manhwafreak.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH)) {
```

Comment:
Name it the same as their website calls itself?

```suggestion
class ManhwaFreakFR : MangaThemesia("ManhwaFreak", "https://manhwafreak.fr", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.ENGLISH)) {
```